eng/common/pipelines/templates/steps/check-spelling.yml

```diff
@@ -5,6 +5,9 @@
 # TargetBranch      - Target ref (e.g. main) to compare to create file change
 #                     list.
 # CspellConfigPath  - Path to cspell.json config location
+#
+# This check recognizes the setting of variable "Skip.SpellCheck"
+# if set to 'true', spellchecking will not be invoked.
```

Comment:
Where do you expect folks to set this variable? In the pipeline or in the ci.yml file?

Reply:
As a pipeline variable.